content/challenges/medium/6 - Capitalize Words/func.js

```diff
@@ -10,5 +10,11 @@
  * @returns {string}
  */
 module.exports = (str) => {
-    // Your code here
+    words = str.split(" ");
+    out_words = []
+    for (i = 0; i < words.length; i++) {
+        let word = words[i]
+        out_words.push(word[0].toUpperCase() + word.slice(1))
+    }
```

Comment:
I believe you can use the map function to make it so you don't need `out_words`. Each iteration of .map({...}) will return the computed value. Similar to a `forEach`.